homie-device/Cargo.toml

```diff
@@ -11,17 +11,15 @@ categories = ["network-programming"]
 
 [dependencies]
 async-channel = "1.6.1"
-futures = "0.3.17"
+futures = "0.3.21"
 local_ipaddress = "0.1.3"
-log = "0.4.14"
-mac_address = "1.1.2"
-rumqttc = "0.10.0"
-tokio = "1.12.0"
-thiserror = "1.0.29"
+log = "0.4.16"
+mac_address = "1.1.3"
+rumqttc = "0.11.0"
```

Comment:
Thanks for doing this.

Andrew points out that you also need to bump all other crates to this version of rumqttc, to fix the build.

I would expect our Cargo.lock to change as well. Make sure you check this in, to make the bin crate builds reproducible.